docs/setup/compile.md

```diff
@@ -73,11 +73,20 @@ For example,
 export SPARK_HOME=$PWD/spark-3.0.1-bin-hadoop2.7
 export PYTHONPATH=$SPARK_HOME/python
 ```
-2. Compile the Sedona Scala and Java code with `-Dgeotools` and then copy the ==sedona-spark-shaded-{{ sedona.current_version }}.jar== to ==SPARK_HOME/jars/== folder.
+2. Put JAI jars to ==SPARK_HOME/jars/== folder.
+```
+export JAI_CORE_VERSION="1.1.3"
```

Comment:
Should we put these jars in geotools-wrapper?

Reply:
These jars are already in geotools-wrapper, so we can instead put the geotools-wrapper jar to the `$SPARK_HOME/jars/` folder, and build spark-shaded jar without `-Dgeotools`. However, this won't be able to test geotools dependency changes such as adding jiffle as a new dependency.

I can update the document to use geotools-wrapper instead of directly using JAI jars, since it is much easier (no need to rebuild with `-Dgeotools` for testing sedona python), and covers most of the cases.

Reply:
This is fine then. We don't need to update this.